presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestQueries.java

```diff
@@ -5719,8 +5720,9 @@ public void testShowTablesFrom()
     @Test
     public void testShowTablesLike()
     {
-        MaterializedResult result = computeActual("SHOW TABLES LIKE 'or%'");
-        assertEquals(result.getOnlyColumnAsSet(), ImmutableSet.of(ORDERS.getTableName()));
+        Set<Object> result = computeActual("SHOW TABLES LIKE 'or%'").getOnlyColumnAsSet();
+        assertThat(result).contains(ORDERS.getTableName());
+        result.forEach(tableName -> assertThat((String) tableName).startsWith("or"));
```

Comment:
Use `allMatch()`:
```java
assertThat(computeActual("SHOW TABLES LIKE 'or%'").getOnlyColumnAsSet())
        .contains(ORDERS.getTableName())
        .allMatch(value -> ((String) value).startsWith("or"));
```
This will give a nice message like
```
java.lang.AssertionError: 
Expecting all elements of:
  <["apple", "banana", "pear", "grape", "cherry"]>
to match given predicate but these elements did not:
  <["banana", "cherry"]>
```

Reply:
I also prefer to just inline the table name `"orders"` since well-known name which will not change and having the constant makes the test easier to read. (with the method call, the reader must wonder if it's doing something special)